easybuild/easyconfigs/h/HTSlib/HTSlib-1.3.1-intel-2016b.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# 1.3.1 modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'HTSlib'
+version = '1.3.1'
+
+homepage = "http://www.htslib.org/"
+description = """A C library for reading/writing high-throughput sequencing data.
+ This package includes the utilities bgzip and tabix"""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [('zlib', '1.2.8')]
+
+sanity_check_paths = {
+    'files': ["bin/bgzip", "bin/tabix", "lib/libhts.%s" % SHLIB_EXT],
```

Comment:
@verdurin any reason `bin/htsfile` is not listed here like it is in the `foss` easyconfig?

The description is also slightly different compared to the `foss` easyconfig, so maybe just copy the `foss` easyconfig to the `intel` easyconfig and fix the `toolchain spec`?